backend/src/main/java/jonaszeihe/ninjin/model/Segment.java

```diff
@@ -0,0 +1,20 @@
+package jonaszeihe.ninjin.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+@Document(collection = "segments")
+public class Segment {
+    @Id
+    private String segmentName;
+    private String segmentContent;
+    private String courseName;
```

Comment:
same as above this time with segment and yes here the usage of course is appropriate as it refers to another class.